fvm/meter/interaction_meter.go

```diff
@@ -117,14 +132,31 @@ func GetStorageKeyValueSizeForTesting(
 	return getStorageKeyValueSize(storageKey, value)
 }
 
-func (m *InteractionMeter) GetStorageUpdateSizeMapForTesting() MeteredStorageInteractionMap {
-	return m.storageUpdateSizeMap
+func (m *InteractionMeter) GetStorageRWSizeMapForTesting() (
+	reads MeteredStorageInteractionMap,
+	writes MeteredStorageInteractionMap,
+) {
+	return m.reads, m.writes
 }
 
+// Merge merges the child interaction meter into the parent interaction meter
+// Prioritise parent reads because they happened first
+// Prioritise child writes because they happened last
```

Comment:
```suggestion
// Prioritise parent reads because they happened first, and child reads will 
//    read from the view (cache) even if they are updated
// Prioritise child writes because they happened last, and parent writes will be overwritten by child writes.
```